pilot/pkg/config/kube/gateway/testdata/backend-tls-policy.yaml.golden

```diff
@@ -1,5 +1,24 @@
 apiVersion: networking.istio.io/v1
 kind: DestinationRule
+metadata:
+  annotations:
+    internal.istio.io/parents: BackendTLSPolicy/default.tls-upstream-echo-https
+  creationTimestamp: null
+  name: echo-https-istio-autogenerated-k8s-gateway
+  namespace: default
+spec:
+  host: echo-https.default.svc.domain.suffix
+  trafficPolicy:
+    portLevelSettings:
+    - port:
```

Comment:
IIUC the input has it on "all ports". and 443

this result has tls only on 443.

My expectation is that 443 would override only for port 443.nkt entirely remove the non-port-specific part? 

LMK if I misunderstood the result or the specifications.